### DISCUSSION TITLE
Could i ask if someone could teach me how to get a server running so i can use stash remotely.

### DISCUSSION THREAD

**Discussion by yujispin990** [stashapp/stash#5492] [Q&A] [answered]
I would like to set up stash so i could use it even when i am not at home.

**Comment by stg-annon** [comment, marked as answer]
related:
https://github.com/stashapp/stash/discussions/2243
https://github.com/stashapp/stash/discussions/3886
https://github.com/stashapp/stash/discussions/4272

**Reply by yujispin990** [reply]
A lot of people are advising against using stash on the web so i will heed said advice and just keep it local. Thank you for the insight!

**Reply by crazycuck** [reply]
I use internally hosted docker version of stash. 

But for remote access I do recommend something like Cloudflare Zero Trust, based on cloudflared tunnel and device+user policies to not directly expose stash or other services.

**Comment by bennibeatnik** [comment]
I recently set up the same scenario and I love it. Instead of Tailscale for the VPN, I went with Twingate. It's incredibly user friendly, faster than cloudflare and tailscale and operates with Zero Trust to open only the ports needed by Stash for a remote connection. For instance, when I turn on Twingate, I browse my Local Lan IPs (192.168.x.x) as if I were at home. Overall the speed is fast enough when Im on WiFi and it's for most intents and purposes the most secure. If you need help setting it up, I can walk you through it step by step. 

https://www.twingate.com/
